src/aws/osml/model_runner/database/feature_table.py

```diff
@@ -72,15 +72,15 @@ def add_features(self, features: List[Feature], metrics: MetricsLogger = None):
         # These records are temporary and will expire 24 hours after creation. Jobs should take
         # minutes to run so this time should be conservative enough to let a team debug an urgent
         # issue without leaving a ton of state leftover in the system.
-        expire_time_epoch_sec = Decimal(int(start_time_millisec / 1000) + (24 * 60 * 60))
+        expire_time_epoch_sec = Decimal(int(start_time_millisec / 1000) + (60 * 60))
```

Comment:
What is the justification for changing the expire time to 1 hour instead of 24? The original thinking behind 24 is that it gives a team time to review the data in-case of a failure/debug. With 1 hour this data may be gone before the team engages on a problem.

Reply:
Good call - the amount of data the table starts accruing during a run in actually huge when put under a very scaled load - and it will start hitting additional provisioning limits to increase it's base size.... I think we say one of the Helix lowside load testing accounts getting up to 500GB? This was an attempt to limit that at first but I think with my latest changes I can revert this and it will at least be able to catch up during the scaling action if/when it happens. But when reviewing the potential runbook with Rob - he/I wasn't sure we would ever need to go back and use these features from the table when triaging something off the top of his head ( or mine for that matter ). But I agree we might want these around for a bit longer.